web/html/src/manager/systems/details/mgr-server-info.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from "react";
+
+import { TopPanel } from "components/panels/TopPanel";
+import { AsyncButton } from "components/buttons";
+
+import { Messages } from "components/messages";
+import { Utils as MessagesUtils } from "components/messages";
+
+import * as Systems from "components/systems/details";
+
+import Network from "utils/network";
+
+// See java/code/src/com/suse/manager/webui/templates/systems/mgr-server.jade
+type Props = {
+  serverId: string;
+  name: string;
+  version: string;
+  reportDbName: string;
+  reportDbHost: string;
+  reportDbPort: number;
+  reportDbUser: string;
+  reportDbLastSynced?: string;
+  isAdmin: boolean;
+  /** Locale of the help links */
+  docsLocale: string;
+};
+
+type State = {
+  messages: any[];
+};
+
+const msgMap = {
+  "Not a system id": t("Not a system id"),
```

Comment:
While using proper strings here will work, I think in most cases we use "key-like" strings that we expect not to change over time a-la `image_overview_not_found`, `base_not_found_or_not_authorized` etc to give some examples from other files. This is not a must, but would be nice for consistency. (You can grep for `const msgMap` to see what I mean.)